dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -297,7 +309,7 @@ end
             'CODE_SIGNING_REQUIRED=NO',
             'CODE_SIGN_IDENTITY=-',
             'EXPANDED_CODE_SIGN_IDENTITY=-',
-            'CONFIGURATION_BUILD_DIR=${objectiveCBuildDirectory.path}',
+            'BUILD_DIR=${objectiveCBuildDirectory.path}',
```

Comment:
`BUILD_DIR` is used to construct `CONFIGURATION_BUILD_DIR`, as well as a few other build settings that were required to move the resource bundle to the right place.

Reply:
Oh, I asked what this was right after you commented, thanks

Reply:
I should have submitted them before requesting review.  You're too speedy 🙂